pkg/modules/replication.go

```diff
@@ -318,7 +320,8 @@ func getReplicaController(op utils.OperatorConfig, cr csmv1.ContainerStorageModu
 	replicaCount := "1"
 	retryMin := "1s"
 	retryMax := "5m"
-	replicaImage := "dellemc/dell-replication-controller:v1.3.1"
+	replicaImage := "dellemc/dell-replication-controller:v1.4.0"
+	replicaInitImage := "dellemc/dell-replication-init:v1.0.0"
```

Comment:
Why the image has to be hardcoded here?

Reply:
I do not know why these were hardcoded here. It should not in my opinion too.
I see that they are used as default vaules and then overwritten from user defined yaml. I see the same for Observability.go as well.